pkg/webhook/webhook.go

```diff
@@ -36,6 +37,23 @@ type combinedDrainData struct {
 	InstanceID string
 }
 
+var instanceEventMap = make(map[string]*monitor.InterruptionEvent)
+var m sync.RWMutex
```

Comment:
1. go-style is to group adjacent variable declarations
2. minor: when using primitive types adding a type alias can clarify an intended field's purpose

```suggestion
type ec2InstanceId = string

var (
    instanceEventMap = make(map[ec2InstanceId]*monitor.InterruptionEvent)
    m sync.RWMutex
)
```